src/VS.ConfigurationManager/Bundle.cs

```diff
@@ -205,5 +199,16 @@ public int Uninstall()
 
             return exitcode;
         }
+        
+        private int ExecuteUninstallProcess(string file)
```

Comment:
{ does not align with the function name.

Reply:
this should be fixed